tests/src/core/physics/Joint.test.ts

```
@@ -5,334 +5,309 @@ import {
+    expect(box2Collider.linearVelocity.y).lessThan(0);
+  });
 
-    joint.stiffness = -10;
-    expect(joint.stiffness).to.eq(-10);
+  it("inertiaScale", function () {
+    engine.sceneManager.activeScene.physics.gravity = new Vector3(0, 0, 0);
+    const box1 = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(0, 1, 0));
+    const box2 = addBox(new Vector3(1, 1, 1), DynamicCollider, new Vector3(1, 1, 0));
+    const fixedJoint = box2.addComponent(FixedJoint);
+    const box2Collider = box2.getComponent(DynamicCollider);
+    const box1Collider = (fixedJoint.connectedCollider = box1.getComponent(DynamicCollider));
```

Biome

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)